### DISCUSSION TITLE
For Juno-G variants without the jumper board

### DISCUSSION THREAD

**Discussion by WampumBiskit** [bjaan/roland-juno-g-display-replacement#5] [Ideas]
amazon carries the 18pin 1.0 pitch flat cable and a converter , so there's no de-soldering anything on the boards.

[https://www.amazon.com/gp/product/B07RT6XWKV/ref=ewc_pr_img_1?smid=A1THAZDOWP300U&th=1](url)

[https://www.amazon.com/gp/product/B0CLYC8TGY/ref=ewc_pr_img_1?smid=A1NR50YASSZD11&psc=1](url)

**Comment by bjaan** [comment]
Good idea! Thank you
